hawc/apps/assessment/management/commands/chemicals.py

```diff
@@ -0,0 +1,43 @@
+import hashlib
+import json
+from typing import Any, Dict
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from hawc.apps.assessment.models import DSSTox
+from hawc.services.epa.dsstox import DssSubstance
+
+
+def _dict_hash(dict: Dict[str, Any]) -> str:
+    """MD5 hash of a dictionary"""
+    dhash = hashlib.md5()
+    encoded = json.dumps(dict, sort_keys=True).encode()
+    dhash.update(encoded)
+    return dhash.hexdigest()
+
+
+class Command(BaseCommand):
```

Comment:
minor changes to this file; i made the main `handle` method basically a way to determine which actions to apply, this is consistent with what we did with the DOI PR.  Also added some text  so we know if it's stuck.

I'm getting a really strange error on my mac, but I didn't see it in other places including our container environment:

```
SSLError: HTTPSConnectionPool(host='comptox.epa.gov', port=443): Max retries exceeded with url:
 /dashboard-api/ccdapp2/chemical-detail/search/by-dsstoxsid?id=DTXSID1037303 (Caused by
 SSLError(SSLError(1, '[SSL: UNSAFE_LEGACY_RENEGOTIATION_DISABLED] unsafe legacy
 renegotiation disabled (_ssl.c:1129)')))
```

I found this link but didn't find much more on it? https://apple.stackexchange.com/questions/435577/